tests/test_utils_types.py

```diff
@@ -22,7 +22,10 @@
 from tower_cli.api import client
 from tower_cli.utils import exceptions as exc, types
 
-from tests.compat import unittest, mock
+try:
+    from tests.compat import unittest, mock
```

Comment:
This is kind of circular with the intent of `compat.py`. No other tests do it this way, and it would be better to keep the same format.